ClassAttendance/src/WebUI/Views/Group/_Group.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using WebUI.Models.ViewModels.GroupModel
+@model GroupViewModel
+@{ 
+    var hasStudents = Model.Students.Any();
+}
+
+<div class="card card-body bg-dark my-auto text-light" style="width: 60%">
+    <h3>@Model.Group.Name</h3>
+    @if (hasStudents)
+    {
+        <h3>Students:</h3>
+        @foreach (var student in Model.Students)
+        {
+           var name = student.FirstName + " " + student.LastName;
```

Comment:
Because of my experience working with MVC5 i guess, i thought that string interpolating is still unavailable to use in views (seems like they use some old C# version in views). Fixed

Reply:
Thank you!